manager/config/integration-tests/deployment_configmap.yaml

```diff
@@ -24,13 +24,14 @@ data:
   #EXTENSIONS_POLICY_MANAGER_NAME: "OPA"
 
   #internal vault related variables
-  VAULT_ADDRESS: "http://vault:8200/"
+  VAULT_ADDRESS: "http://vault.m4d-system:8200/"
```

Comment:
I don't see here any cluster reference. Does this address work when connecting from another cluster?

Reply:
In deployment mode of multi-cluster setup this address should be the address of the Ingress to Vault. So it should be set based on how the clusters were configured - the Ingress/Route address for multi-cluster (we do not need Ingress in single cluster) and for single cluster vault.m4d-system is enough. I will need to clarify that in the the documentation.